Resources/Prototypes/_Goobstation/Entities/Structures/Doors/Airlocks/airlocks.yml

```
@@ -25,4 +28,33 @@
+      board: [ DoorElectronicsAtmospherics ]
+  - type: Sprite
+    sprite: _Goobstation/Structures/Doors/Airlocks/Glass/atmossealed.rsi
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.49,-0.49,0.49,0.49"
+        density: 100
+        mask:
+          - FullTileMask
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)